### PR TITLE
Fix/refactor light

### PR DESCRIPTION
**Pull request by benmaidel**
this is needed for new cob_light features.
connected to https://github.com/ipa320/cob_driver/pull/262

### CODE REVIEW

**Comment by fmessmer** [comment]
please keep in mind that there is a PR pending for cob_teleop in https://github.com/ipa320/cob_command_tools/pull/132 which restructures the node significantly...i.e. it rewrites the code for  calling the `set_light` action
Could we merge https://github.com/ipa320/cob_command_tools/pull/132 first and then re-apply the modifications introduced in this PR again to the updated `indigo_dev`?

File: cob_script_server/src/simple_script_server/simple_script_server.py [hunk 1]
```diff
@@ -85,7 +85,7 @@
 # care-o-bot includes
 from cob_sound.msg import *
 from cob_script_server.msg import *
-from cob_light.msg import LightMode, SetLightModeGoal, SetLightModeAction
+from cob_light.msg import LightMode, LightModes, SetLightModeGoal, SetLightModeAction
```
Comment (fmessmer) [inline cob_script_server/src/simple_script_server/simple_script_server.py:88]: `LightModes` is node used here...but we

File: cob_script_server/src/simple_script_server/simple_script_server.py [hunk 2]
```diff
@@ -742,10 +742,13 @@ def set_light(self,component_name,parameter_name,blocking=False):
 
 		mode = LightMode()
 		mode.mode = 1
```
Comment (fmessmer) [inline cob_script_server/src/simple_script_server/simple_script_server.py]: ...should use it here!

File: cob_teleop/ros/src/cob_teleop.cpp [hunk 3]
```diff
@@ -285,15 +285,16 @@ sensor_msgs::JoyFeedbackArray CobTeleop::switch_mode(){
 
   light.mode = 2;
```
Comment (fmessmer) [inline cob_teleop/ros/src/cob_teleop.cpp]: the according `LightMode` const should be used here...

**Comment by fmessmer** [comment]
@ipa-bnm please merge with ipa320/indigo_dev

**Comment by benmaidel** [comment]
travis failes because message description from here https://github.com/ipa320/cob_driver/pull/262 is missing

**Comment by benmaidel** [comment]
ready to merge

**Comment by fmessmer** [comment]
FR ok

**Comment by fmessmer** [comment]
travis is happy

**Pull request merged**